enable.csh.in

```diff
@@ -36,3 +37,8 @@ cat <<EOF
 * you can change your login shell from csh to bash.                        *
 ****************************************************************************
 EOF
```

Comment:
should this EOF be here still?

Reply:
That is a political decision - whether we should support csh fully, or immediately set it on the depreciation bandwagon.